src/adapter-types/src/dyncfgs.rs

```diff
@@ -19,6 +19,12 @@ pub const ALLOW_USER_SESSIONS: Config<bool> = Config::new(
     "Whether to allow user roles to create new sessions. When false, only system roles will be permitted to create new sessions.",
 );
 
+pub const ENABLE_0DT_DEPLOYMENT: Config<bool> = Config::new(
+    "enable_0dt_deployment",
+    true,
```

Comment:
Almost didn't catch this! But `true` is probably not the default we want right now? 😅